Refactory on error handling of SQLBatch

EduFrazao opened the pull request:
- All errors refactored to be returned on the callback
   message instead of throwing it.

ospfranco commented:
oops, I deleted my comment because non-execution errors were still throwing errors in the other functions, but you went ahead and implemented it anyways. I'll try to review later, thanks!

EduFrazao commented:
> oops, I deleted my comment because non-execution errors were still throwing errors in the other functions, but you went ahead and implemented it anyway. I'll try to review it later, thanks!

No problem!! Thank you @ospfranco
